pkgs/applications/networking/browsers/google-chrome/default.nix

```diff
@@ -142,8 +142,9 @@ in stdenv.mkDerivation {
     makeWrapper "$out/share/google/$appname/google-$appname" "$exe" \
       --prefix LD_LIBRARY_PATH : "$rpath" \
       --prefix PATH            : "$binpath" \
-      --prefix XDG_DATA_DIRS   : "$XDG_ICON_DIRS:$GSETTINGS_SCHEMAS_PATH:${addOpenGLRunpath.driverLink}/share" \
-      --add-flags ${escapeShellArg commandLineArgs}
+      --prefix XDG_DATA_DIRS   : "$XDG_ICON_DIRS:$GSETTINGS_SCHEMAS_PATH" \
+      --add-flags ${escapeShellArg commandLineArgs} \
+      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--enable-features=UseOzonePlatform --ozone-platform=wayland}}"
```

Comment:
`commandLineArgs` exists, might be better to use that?

Reply:
This one is weird, and I guess it might explain why it's not broken (IIUC one of the comments).
It only seems to need `escapeShellArg` once for `commandLineArgs` instead of twice, is there a second escaping elsewhere?

The only difference I can see is this is in `installPhase` but `chromium` is in `buildCommand`?